pom.xml

```diff
@@ -212,6 +127,13 @@
     <maven.build.timestamp.format>yyyyMMdd-HHmm</maven.build.timestamp.format>
     <jacoco-maven-plugin.version>0.7.6.201602180812</jacoco-maven-plugin.version>
     <mockito.version>1.10.19</mockito.version>
+    
+    <final.war.name>cbioportal-${project.version}</final.war.name>
+
+	<db.test.driver>com.mysql.jdbc.Driver</db.test.driver>
+	<db.test.url>jdbc:mysql://127.0.0.1:3306/cgds_test</db.test.url>
+    <db.test.username>cbioportal</db.test.username>
+    <db.test.password>cbioportal</db.test.password>
```

Comment:
Thanks @morungos. That clears things up. I will try to add that to the guidelines for reviewing.

I'm guessing that 90% of current config in `.properties` should actually be runtime.

For runtime, if we are using a property file outside the war, I propose we follow `env variable > properties file > default in pom`.